server/templates/success.html

```diff
@@ -0,0 +1,6 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h1>Registration Successful!</h1>
```

Comment:
The membership number can be displayed here....

Reply:
currently not sure on how to do this

Reply:
Posted a note below, in the views.py